sys-libs/libcxx/libcxx-18.1.8.ebuild

```diff
@@ -76,12 +76,15 @@ test_compiler() {
 src_configure() {
 	llvm_prepend_path "${LLVM_MAJOR}"
 
+	local sysroot
```

Comment:
Nit: naming it `sysroot` is confusing (I mean, sysroot normally goes *before* `EPREFIX`).

Reply:
I meant the standard `${ESYSROOT}` variable expands to `${SYSROOT}${EPREFIX}`.

Reply:
Oh, sorry, I think what you meant is the `${ESYSROOT}...` pattern, which is present in the same file. I'm convinced now, the variable naming `sysroot` is definitely confusing in that context.

To make it less confusing, what about:

```
	local prefix=${EPREFIX}
	target_is_not_host && prefix+=/usr/${CTARGET}

	local libdir=$(get_libdir)
	local mycmakeargs=(
                [...]
		-DCMAKE_INSTALL_PREFIX="${prefix}/usr"
                [...]
        )
```

Does it look good to you?

Reply:
Let's make it `install_prefix`.

Reply:
Done